### ISSUE TITLE
Random 404 errors

### ISSUE THREAD

**Issue by brianmay** [helm/chart-releaser-action#84]
https://github.com/brianmay/charts/runs/2899574615

```
Looking up latest tag...
Discovering changed charts since 'penguin-memories-0.0.2'...
Installing chart-releaser...
Adding cr directory to PATH...
Packaging chart 'charts/scrooge'...
Successfully packaged chart in /home/runner/work/charts/charts/charts/scrooge and saved it to: .cr-release-packages/scrooge-0.0.2.tgz
Releasing charts...
Updating charts repo index...
Using existing index at .cr-index/index.yaml
Error: GET https://api.github.com/repos/brianmay/charts/releases/tags/scrooge-0.0.2: 404 Not Found []
```

I suspect there might a race condition with github. We create the tag github, but github can't find it immediately, due to caching or something.

Have had this happen twice today.

**Comment by davidkarlsen** [comment]
What version do you use? Retries were added some time ago.

**Comment by brianmay** [comment]
This is version 1.1.0.

Have now setup dependabot on this repo.

Going to assume this is fixed in 1.2.1, closing.

Thanks for prompting me to upgrade.
